### PR TITLE
About Us Page Refactor

### PR DESCRIPTION
**Pull request by AlisaSev**
image
{07433291-63C5-411E-BB4E-2E05D2F9D7A7}

In this task, I built a system to fetch developer data from our SQL database and display it on the front end:

- SQL: Created the SQL tables and stored procedures.

- .NET: Developed all the necessary components, including models and services to handle developer data. Also configured the CORS policy to allow cross-origin requests between the front end and back end.

- Front-End: Added the React components to display the developer information, using AJAX calls to fetch data from the backend.

- Styling: Adjusted the page margins for a better layout and fixed the header so that it remains static and does not shrink.

### CODE REVIEW

File: dotnet/LADP_EFC/Data/DataContext.cs [hunk 1]
```diff
@@ -87,6 +91,18 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
...
+            modelBuilder.Entity<Developer>(entity =>
+            {
+                entity.ToTable("DevelopersInfo");
+                entity.HasKey(e => e.DeveloperID).HasName("PK_DevelopersInfo");
+                entity.Property(e => e.DeveloperID).HasColumnName("DeveloperID").HasColumnType("int");
+                entity.Property(e => e.Name).IsRequired().HasMaxLength(255).HasColumnName("Name").HasColumnType("nvarchar(255)");
+                entity.Property(e => e.Role).IsRequired().HasMaxLength(100).HasColumnName("Role").HasColumnType("nvarchar(100)");
+                entity.Property(e => e.PictureUrl).HasColumnName("PictureUrl").HasColumnType("nvarchar(MAX)");
+                entity.Property(e => e.LinkedInUrl).HasColumnName("LinkedInUrl").HasColumnType("nvarchar(MAX)");
+                entity.Property(e => e.GithubUrl).HasColumnName("GithubUrl").HasColumnType("nvarchar(MAX)");
```
Comment (jjcanell77) [inline dotnet/LADP_EFC/Data/DataContext.cs]: These column types should match what you have in the database `nvarchar(255)` to avoid errors. `nvarchar(MAX)` should be avoided because of the amount of space it takes in the database. The description in FoodResource will need to be addressed eventually as well in another PR as it uses `nvarchar(MAX)`  as well.
image

**Review by jjcanell77 — CHANGES_REQUESTED** [review]
Just that small change but looks good. I am interested to look more into CORS.

File: dotnet/LADP_EFC/Data/DataContext.cs (hunk 1 above)
Comment (na106) [reply dotnet/LADP_EFC/Data/DataContext.cs]: Everything else looks good. I think what Jesse said is all that has to be updated.

File: .vs/LADP.generated/v17/DocumentLayout.json
Comment (jmedina1329) [inline .vs/LADP.generated/v17/DocumentLayout.json:1]: Unfortunately we have the same issue here with all these random files in your PR. Please link up with the team and let's find time to get this resolved so we can merge your work.

**Review by jmedina1329 — CHANGES_REQUESTED** [review]
